service.go

```diff
@@ -90,6 +90,17 @@ func (s *service) IssueCert(ctx context.Context, entityID string, ipAddrs []stri
 		return "", ErrRootCANotFound
 	}
 
+	// Parse the TTL if provided, otherwise use the default certValidityPeriod
+	var validity time.Duration
+	if ttl != "" {
+		validity, err = time.ParseDuration(ttl)
+		if err != nil {
+			return "", errors.Wrap(ErrMalformedEntity, err)
+		}
+	} else {
+		validity = certValidityPeriod
+	}
```

Comment:
Use full stop in the comment and update to something like:
```suggestion
// Parse the TTL if provided, otherwise use the default certValidityPeriod.
	validity := certValidityPeriod
	if ttl != "" {
		if validity, err = time.ParseDuration(ttl); err != nil {
			return "", errors.Wrap(ErrMalformedEntity, err)
		}
	} 
```